examples/widgets/search_entry.nim

```diff
@@ -73,7 +74,7 @@ method view(app: AppState): Widget =
             app.text = ""
             app.filteredItems = app.items
           
-      ScrolledWindow:
+      ScrolledWindow as app.keyCaptureRef:
```

Comment:
Turns out this will not fill the keyCaptureRef if you use `ScrolledWindow() as app.keyCaptureRef`. Which must be fixed.